### PR TITLE
Added reject with only a Class argument

### PR DESCRIPTION
**Pull request by tivervac**
This API exists for filter, but not for reject

### CODE REVIEW

**Comment by cdietrich** [comment]
please have a look at the build errors

**Comment by tivervac** [comment]
@cdietrich I'm wondering if we should filter the nulls out of the result, any thoughts?

**Comment by cdietrich** [comment]
Good question. I don’t know how others eg guava behave in that respect @szarnekow what do you think

**Comment by cdietrich** [comment]
Jenkins still shows test errors

**Comment by tivervac** [comment]
I'll wait for a response to fix that test. The response determines how I should fix it

**Comment by szarnekow** [comment]
I think leaving null in is ok for the method. LGTM

**Comment by tivervac** [comment]
Alright, I'll fix the failing test tomorrow

File: org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/IterableExtensions.java [hunk 1]
```diff
@@ -271,6 +271,25 @@
...
 	}
 
+	/**
+	 * Returns the elements of {@code unfiltered} that are not instanceof {@code type}. The resulting iterable's iterator does not
+	 * support {@code remove()}. The returned iterable is a view on the original elements. Changes in the unfiltered
+	 * original are reflected in the view.
+	 *
+	 * @param unfiltered
+	 *            the unfiltered iterable. May not be <code>null</code>.
+	* @param type
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/IterableExtensions.java]: can you add a may no be null

**Comment by cdietrich** [comment]
can you please rebase and squash your commits

**Comment by tivervac** [comment]
The issue was with GWT, apparently isInstance is not supported, should be fixed now.

**Comment by cdietrich** [comment]
thx @tivervac

**Pull request merged**